Bearing/BearingTitlePageBuilder.cpp

```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////
 // BEToolbox
-// Copyright � 1999-2023  Washington State Department of Transportation
+// Copyright � 1999-2022  Washington State Department of Transportation
```

Comment:
Uncomment line 95 so the is an image on the printed report title page.

Reply:
Done. I still need to store an image.